config.js

```diff
@@ -0,0 +1,75 @@
+// NOTE: Neither of these files are committed and for good reason.
```

Comment:
will configuration json (ie chesster.json and tesster.json) still be something loaded from the file as well, or just a config.js.

seems we shouldnt hard code this right?